lib/compiler_rt/gehf2.zig

```diff
@@ -22,10 +27,10 @@ pub fn __gthf2(a: f16, b: f16) callconv(.C) i32 {
     return __gehf2(a, b);
 }
 
-fn __aeabi_fcmpge(a: f16, b: f16) callconv(.AAPCS) i32 {
+fn __aeabi_hcmpge(a: f16, b: f16) callconv(.AAPCS) i32 {
     return @boolToInt(comparef.cmpf2(f16, comparef.GE, a, b) != .Less);
 }
 
-fn __aeabi_fcmpgt(a: f16, b: f16) callconv(.AAPCS) i32 {
+fn __aeabi_hcmpgt(a: f16, b: f16) callconv(.AAPCS) i32 {
```

Comment:
why did you rename the functions?

Reply:
__aeabi_fcmpgt has f32 as input sizes (f for float). However, we have here half floats (f16).

Reply:
Actually the functions should be removed from gehf2.zig, since __aeabi_hcmpgt and __aeabi_hcmpge don't exist. __aeabi_fcmpgt and __aeabi_fcmpge are already defined and exported in gesf2.zig, so we shouldn't export them in gefh2.zig. I renamed the function, since I thought that cmpgt and cmpge functions existed for half-precision-floats, but obviously they don't. I'm sorry, I should have done more research to validate that these functions exist. I will add a new commit that fully removes the functions.

Reply:
Also my bad to not take a closer look initially. I was abit in a hurry.